### PR TITLE
chore: fix vuetify package build on Windows

### PR DESCRIPTION
**Pull request by userquin**
## Description

This PR fixes building vuetify package on Windows.

## Markup:

NA

### CODE REVIEW

**Comment by userquin** [comment]
I need to fix this:

imagen

File: packages/vuetify/build/rollup.config.mjs [hunk 1]
```diff
@@ -290,10 +288,11 @@ export default [
             source: variables.map(id => {
               return `@forward '` + path.join(
                 '../lib',
-                path.relative(fileURLToPath(new URL('../src', import.meta.url)), id),
+                // Normalize Windows Paths
```
Comment (userquin) [inline packages/vuetify/build/rollup.config.mjs:291]: remove this or move it

File: packages/vuetify/build/rollup.config.mjs [hunk 2]
```diff
@@ -171,10 +168,11 @@ export default [
...
```
Comment (userquin) [inline packages/vuetify/build/rollup.config.mjs:171]: remove this or move it

**Comment by KaelWD** [comment]
Does `upath` remove the need for all these `.replace(/\\/g, '/')`?

**Comment by userquin** [comment]
> Does `upath` remove the need for all these `.replace(/\\/g, '/')`?

I guess no, we are using fs paths functions like join or relative. I can check it later.

**Pull request merged**